### PR TITLE
Folder structure

### PR DESCRIPTION
**Pull request by korsa719**

### CODE REVIEW

**Comment by korsa719** [comment]
[Create one instance of class for Controller-Service-DAO layer](https://trello.com/c/z141yscZ/23-create-one-instance-of-class-for-controller-service-dao-layer)

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (zozich) [inline .idea/misc.xml]: Why is this file here? All .idea folder should be included in .gitignore and should not appear on GitHub

File: text-messenger-client/src/index.js [hunk 2]
```diff
@@ -5,4 +5,4 @@ import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
 ReactDOM.render(<App />, document.getElementById('root'));
-registerServiceWorker();
+// registerServiceWorker();
```
Comment (zozich) [inline text-messenger-client/src/index.js]: Why?
